### PR TITLE
update webhook deploy comment

### PR DESCRIPTION
**Pull request by mfrancisc**
## Description
Update the `WebhookConfig.Deploy` field which now is used in a different way. Based on the changes [in this PR](https://github.com/codeready-toolchain/member-operator/pull/562) and related ones, the webhook will still be deployed but deleted afterwords.

Those PRs will be merged only after the new behavior is merged to master. 

## Checks
1. Did you run `make generate` target? **yes**

2. Did `make generate` change anything in other projects (host-operator, member-operator)? **yes**

3. In case of **new** CRD, did you the following? **N/A**
    - make/generate.mk in this repository
    - `resources/setup/roles/host.yaml` in the sandbox-sre repository
    - `PROJECT` file: https://github.com/codeready-toolchain/host-operator/blob/master/PROJECT
    - `CSV` file: https://github.com/codeready-toolchain/host-operator/blob/master/config/manifests/bases/host-operator.clusterserviceversion.yaml

4. In case other projects are changed, please provides PR links.
    - host-operator: https://github.com/codeready-toolchain/host-operator/pull/1036
    - member-operator: https://github.com/codeready-toolchain/member-operator/pull/572

### CODE REVIEW

File: api/v1alpha1/memberoperatorconfig_types.go [hunk 1]
```diff
@@ -155,7 +155,8 @@ type ToolchainClusterConfig struct {
 // Defines all parameters concerned with the Webhook
 // +k8s:openapi-gen=true
 type WebhookConfig struct {
-	// Defines the flag that determines whether to deploy the Webhook
+	// Defines the flag that determines whether to deploy the Webhook.
+	// If the deploy flag is disabled, the Webhook is deployed and deleted immediately after by the memberoperatorconfig controller.
```
Comment (rajivnathan) [inline api/v1alpha1/memberoperatorconfig_types.go:159]: It kind of sounds like we're saying the webhook will be deployed and then deleted again. How about rewording it like this to make it a bit more clear?

```suggestion
	// If the deploy flag is set to False and the Webhook was deployed previously it will be deleted by the memberoperatorconfig controller.
```

**Review by rajivnathan — APPROVED** [review]
Thanks for updating the comment 👍 Just made a small suggestion regarding the wording

File: api/v1alpha1/memberoperatorconfig_types.go (hunk 1 above)
Comment (mfrancisc) [reply api/v1alpha1/memberoperatorconfig_types.go:159]: yup, makes more sense indeed!

**Review by rajivnathan — APPROVED** [review]
